phpstan.neon

```diff
@@ -0,0 +1,31 @@
+parameters:
+    level: 10
+    tmpDir: build/phpstan.cache
+    paths:
+        - src
+        - tests
+    ignoreErrors:
+        - message: '#(Array shape contains unspecified keys|Parameter .+ of method .+ contains unspecified keys|Array shape is not compatible with|Property .+ type with generic class .+ does not specify its types|Cannot access offset .+ on array\{.+\}|array\{.+\} does not accept key .+|Array has no value type specified in iterable type array)#'
```

Comment:
It's now possible to use identifiers ignore errors. don't know if it's done on the error message here on purpose or because it's used to be the only option ( https://phpstan.org/blog/phpstan-1-11-errors-identifiers-phpstan-pro-reboot )